backend/api_handler.py

```diff
@@ -115,6 +115,7 @@ def handler(event, context):
 
     if 'authorizer' in event['requestContext']:
         username = event['requestContext']['authorizer']['claims']['email']
+        email_id = event['requestContext']['authorizer']['claims']['email']
```

Comment:
Why don't we use the username? It seems to be have the same value as the email. Is this to make it clear that we the variable contains the email?

Reply:
Yes, this was to make it clear that the variable contains email_id. As we are fetching the email attribute from the authorizer claim. I didn't want to refactor the username field and change it everywhere so I created a new field